apps/web/src/components/bid-activity/bid-activity.tsx

```diff
@@ -82,10 +78,10 @@ export default function BidActivity({
           const avatar = avatars[bid.externalId]
           const createdAtDateTime = new Date(bid.createdAt)
           const meetsReservePrice =
-            reservePrice && isGreaterThanOrEqual(bid.amount, reservePrice)
+            !!reservePrice && isGreaterThanOrEqual(bid.amount, reservePrice)
```

Comment:
When the reserve price wasn't included in the pack template, "0" would display in the activity table. This fixes that.